dev/make-distribution.sh

```diff
@@ -163,7 +163,7 @@ echo -e "\$ ${BUILD_COMMAND[@]}\n"
 rm -rf "$DISTDIR"
 mkdir -p "$DISTDIR/jars"
 echo "Spark $VERSION$GITREVSTRING built for Hadoop $SPARK_HADOOP_VERSION" > "$DISTDIR/RELEASE"
-echo "Build flags: $@" >> "$DISTDIR/RELEASE"
+echo "Build flags: "$@"" >> "$DISTDIR/RELEASE"
```

Comment:
I'm not sure this does what you think -- doesn't this actually leave `$@` outside quotes?
The rest looks OK but I don't know when the script would have an argument with spaces? Look how it's used.